modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -8,17 +8,146 @@ label:AuraDB-Business-Critical[]
 
 == SSO levels
 
-Organization admins can configure organization level SSO (org SSO) and tenant level SSO (tenant SSO). 
+Organization admins can configure organization level SSO (org SSO) and tenant level SSO (tenant SSO).
 SSO is a log-in method and access, roles, and permissions are dictated by role-based access control (RBAC).
 
-* *Org SSO:* Allows org admins to restrict how users log in when they are trying to access the org. 
+* *Org SSO:* Allows org admins to restrict how users log in when they are trying to access the org.
 Access beyond log-in is managed via RBAC.
 
-* *Tenant level SSO:*  Impacts new database instances created within that tenant. 
-It ensures users logging in with SSO have access to the database instances within the tenant. 
-It depends on RBAC if the user can access and view or modify data within the database instances themselves. 
+* *Poject-level SSO:*  Impacts new database instances created within that project.
+It ensures users logging in with SSO have access to the database instances within the project.
+It depends on RBAC if the user can access and view or modify data within the database instances themselves.
 For this level, the role mapping may be used to grant users different levels of access based on a role in their identity provider (IdP).
-It *does not* give access to edit the tenant settings, for example to edit the tenant name, network access, or to edit the instance settings such as to rename an instance, or pause and resume.
+It *does not* give access to edit the project settings, for example to edit the project name, network access, or to edit the instance settings such as to rename an instance, or pause and resume.
+
+=== SSO Org level roles
+
+The following roles are available at the org level and these are assigned via invitation:
+
+* Owner
+* Admin
+* Member
+
+:check-mark: icon:check[]
+
+.Roles
+[opts="header",cols="3,1,1,1"]
+|===
+| Capability
+| Owner
+| Admin
+| Member
+
+| List org
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| List projects
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| Update org
+| {check-mark}
+| {check-mark}
+|
+
+| Add projects
+| {check-mark}
+| {check-mark}
+|
+
+| List existing SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Add SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| List project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Update project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Delete project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Invite non-owner users to org
+| {check-mark}
+| {check-mark}
+|
+
+| List users
+| {check-mark}
+| {check-mark}
+|
+
+| List roles
+| {check-mark}
+| {check-mark}
+|
+
+| List members of a Project
+| {check-mark}
+| {check-mark}
+|
```

Comment:
Admins can't do anything about project members unless they are part of the project